task_checker.py

```diff
@@ -0,0 +1,40 @@
+import pickle
```

Comment:
purpose is to check to see that at least one photo from a photo set has been migrated.  It does this by creating a list of URLs from all of the completed tasks.  Then for each photoset, get the url of the first photo and see if it exists as a completed task.